### PR TITLE
Do a full browser reload on addition/deletion of items to www directory.

### PR DESCRIPTION
**Pull request by omefire**

### CODE REVIEW

**Comment by omefire** [comment]
Related issue: https://github.com/omefire/cordova-plugin-livereload/issues/15

**Comment by omefire** [comment]
@riknoll, @nikhilkh, @csantanapr ..
Please, review

File: lib/postPrepareHook.js [hunk 1]
```diff
@@ -85,7 +81,17 @@ module.exports = function(context) {
...
+                        lr.browserSync.reload(file);
+                        return Q();
+                    }
+
+                    // If new files got added or deleted, reload the whole app instead of specific files only
+                    // e.g: index.html references a logo file 'img/logo.png'
+                    // deleting the 'img/logo.png' file will trigger a reload that will remove it from the rendered app
+                    // likewise, adding the 'img/logo.png' file will trigger it to be shown on the app
+                    lr.browserSync.reload();
+                    return Q();
```
Comment (nikhilkh) [inline lib/postPrepareHook.js:94]: Nit: Why return Q()? just return should have the same effect.

**Comment by nikhilkh** [comment]
LGTM

**Pull request merged**